modules/lib/src/main/resources/assets/js/app/view/ContentItemPreviewPanel.ts

```diff
@@ -441,4 +366,40 @@ export class ContentItemPreviewPanel
         return this.mask.isVisible();
     }
 
+    getActions(): Action[] {
+        return [
+            ...super.getActions(),
+            this.getPreviewAction()
+        ];
+    }
+
+    private getPreviewAction(): WidgetPreviewAction {
+        return this.getToolbar().getPreviewAction();
+    }
+
+    private isResponseOk(response: Response, previewWidget: Widget) {
+        const isAuto = previewWidget?.getWidgetDescriptorKey().getName() === 'preview-automatic';
+        return response.ok || !isAuto && response.status !== StatusCode.I_AM_A_TEAPOT;
+    }
+
+    private applyImageStyles(frameWindow: Window) {
+        const body = frameWindow.document.body;
+        if (body) {
+            body.style.display = 'flex';
+            body.style.justifyContent = 'center';
+            body.style.alignItems = 'center';
+        }
+
+        let img: HTMLImageElement | SVGElement = frameWindow.document.querySelector('svg');
+        if (img) {
+            img.style.margin = '0 auto';
+            img.style.height = '100%';
+        } else {
+            img = frameWindow.document.querySelector('body > img');
+        }
+        if (img) {
+            img.style.maxWidth = '100%';
+            img.style.maxHeight = '100%';
+        }
```

Comment:
Can't we set this via CSS? If not, then we need to do `new ElementHelper(body)`, then use setters from ElementHelper (`setDisplay`, `setMaxWidth` etc)